optuna/integration/tensorboard.py

```diff
@@ -0,0 +1,155 @@
+import optuna
+from optuna import type_checking
+import os
+
+if type_checking.TYPE_CHECKING:
+    from typing import Any  # NOQA
+    from typing import Dict  # NOQA
+    from typing import Optional  # NOQA
+    from typing import Tuple  # NOQA
```

Comment:
Please import build-in libraries at first, then import third-party libraries.

```suggestion
import os
from typing import Dict

import optuna
```